dev.fredol.open-tv.yaml

```diff
@@ -0,0 +1,1106 @@
+id: dev.fredol.open-tv
+
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+
+command: open_tv
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --filesystem=host:ro
+  - --socket=pulseaudio
+  - --filesystem=/tmp:rw
+  - --filesystem=xdg-pictures:rw
+  - --filesystem=xdg-videos:rw
+  - --filesystem=home/.var/app/io.mpv.Mpv/config/mpv/scripts:create
```

Comment:
definitely NOT. 

First it's invalid.

Second, this application is not `io.mpv.Mpv`

Reply:
Removed it